app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserListAdapter.java

```diff
@@ -679,27 +707,28 @@ private void loadFolder(final File folder, final File show) {
         final File toShow = show == null ? _fileToShowAfterNextLoad : show;
         _fileToShowAfterNextLoad = null;
 
+        _currentFolder = folder;
```

Comment:
We set this early and in the wrapping sync function. Prevents bugs from preventing this from being set.